src/utils/index.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs')
+const isPng = require('is-png')
+const crypto = require('crypto')
+
+const _validatePng = function (value) {
```

Comment:
Don't underscore it if we are exporting it. (And if it is private, use double unders...)